### ISSUE TITLE
Protobuf Warnings

### ISSUE THREAD

**Issue by chiragvels** [googleads/google-ads-php#500]
**What versions of the client library and Google Ads API are you using?**
V6

**What environment are you using?**
C implementation of `protobuf` - 3.14.0
gRPC 1.33.1
PHP 7.2.16

**Actions taken**
I need to pause placements/mobile app/youtube channel
for that I 

```
        $adGroupCriterion = new AdGroupCriterion([
            'resource_name' => ResourceNames::forAdGroupCriterion(
                $customerId,
                $adGroupId,
                $criterionId
            ),
             'status' => AdGroupCriterionStatus::PAUSED
       ]);
        $adGroupCriterionOperation = new AdGroupCriterionOperation();
        $adGroupCriterionOperation->setUpdate($adGroupCriterion);
        $adGroupCriterionOperation->setUpdateMask(FieldMasks::allSetFieldsOf($adGroupCriterion));
```

**Expected result**
Placement should be pause

**Actual result**
Placement is paused but with some warning like
WARNING: file was already added

**Comment by PierrickVoulet** [comment]
Hi @chiragvels - Thank you for the feedback. I created an [issue](https://github.com/protocolbuffers/protobuf/issues/8124) on the Protobuf project as these warnings are unexpected.

**Comment by PierrickVoulet** [comment]
Update: Protobuf changed their logic so that those messages are now logged as [`E_USER_WARNING`](https://www.php.net/manual/en/errorfunc.constants.php) with actionable information.

**Comment by PierrickVoulet** [comment]
The Protobuf team just made a fix to mitigate the issue https://github.com/protocolbuffers/protobuf/pull/8720. It should be [released](https://github.com/protocolbuffers/protobuf/releases/) in the next version of Protobuf.

**Comment by PierrickVoulet** [comment]
Update: This issue was fixed in the version [3.18.0](https://github.com/protocolbuffers/protobuf/releases/tag/v3.18.0) of `protobuf`.
